src/org/opensolaris/opengrok/web/HtmlConsts.java

```diff
@@ -33,4 +33,13 @@
     public static final String SPAN_N = "<span class=\"n\">";
     public static final String SPAN_S = "<span class=\"s\">";
```

Comment:
will these be replaced by below ?

Reply:
Yes, these public constants later could be mostly relocated and privatized.

(ZSPAN would survive here, and the others would continue to be useful — privately — in web/Util.)

Reply:
ok, so I suppose this can happen next, I will merge this, let's move forward
thank you Chris!

Reply:
My pleasure, @tarzanek !

I have a queue of pull requests for language tests and fixes that I'll start submitting now.